src/Compilers/Core/Portable/Desktop/DesktopAssemblyIdentityComparer.Fx.cs

```diff
@@ -109,7 +109,9 @@ public override int GetHashCode()
                 string name,
                 ImmutableArray<byte> publicKeyToken,
                 AssemblyVersion versionLow,
+#pragma warning disable IDE0060 // Remove unused parameter
```

Comment:
:memo: The code that calls this (a collection initializer below, `s_arRetargetPolicy`) is copied from some other code base. I prefer leaving it as-is.